graphs/fleurys_algorithm.py

```diff
@@ -0,0 +1,110 @@
+from collections import defaultdict
+
+
+class Graph:
+
+    def __init__(self, vertices):
+        self.v = vertices
+        self.graph = defaultdict(list)
+
+    def addEdge(self, u, v):
```

Comment:
```suggestion
    def add_edge(self, universal: int, victorious: float) -> list[int]:
```
Don't force your reader to guess.  Type hints are required in this repo so the caller knows what datatypes the functions expect and what datatype they return.

Single -letter variable names for function parameters is the way that programs were written in the 1950's and 60's.  In modern times, we no longer force users to guess what variables mean.  Instead, we give them self-documenting names.

Reply:
okay thank you